.github/workflows/main.yml

```diff
@@ -319,9 +319,11 @@ jobs:
             continue-on-error: true
           - partner: model-fragments
             continue-on-error: true
+          - partner: ember-observer
+            continue-on-error: true
```

Comment:
This is currently failing.  Ran locally to confirm.  I've seen this error before but not sure exactly what is going on now.

![Screen Shot 2021-08-08 at 2 13 43 PM](https://user-images.githubusercontent.com/7374640/128643216-a595e41d-ceaf-4a6e-b7d5-6923067348f2.png)